NOTICE.txt

```diff
@@ -4,3 +4,19 @@ Copyright 2011-2016 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+******************
+SIL OFL 1.1
+******************
+
+The following binary components are provided under the SIL Open Font License 1.1
```

Comment:
Looking at the text at http://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web, i feel it shouldn't require mention in NOTICE.  Same with CC BY-SA as well. 

But no harm in mentioning, as NOTICE is already a light one.